generators/app/__snapshots__/generator.spec.ts.snap

```diff
@@ -835,6 +859,9 @@ exports[`generator - app with default config should match snapshot 1`] = `
   "skipJhipsterDependencies": undefined,
   "skipServer": undefined,
   "skipUserManagement": false,
+  "sonarName": undefined,
+  "sonarOrga": undefined,
```

Comment:
Should this be `sonarOrg`?

Reply:
Was like this previously, didn't changed anything here